examples/custom-server-fastify/server.js

```diff
@@ -1,4 +1,4 @@
-const fastify = require('fastify')({ logger: { level: 'error' } })
+const fastify = require('fastify')({ logger: { level: 'error' },pluginTimeout:0 }) 
```

Comment:
This might fix the linting issue.

> Code style issues found in the above file(s). Forgot to run Prettier?

```suggestion
const fastify = require('fastify')({ logger: { level: 'error' }, pluginTimeout: 0 })
```